pkg/operators/filter/filter.go

```diff
@@ -73,6 +73,7 @@ func (f *filterOperator) InstanceParams() api.Params {
     field<value      - matches, if the content of field is less than the value
     field~value      - matches, if the content of field matches the regular expression 'value'
                  see [https://github.com/google/re2/wiki/Syntax] for more information on the syntax
+  Multiple filters can be combined using a comma: field1==value1,field2==value2
```

Comment:
Does [cmd/common/registry.go#L288-L296](https://github.com/inspektor-gadget/inspektor-gadget/blob/main/cmd/common/registry.go#L288-L296) need similar documentation too?

Reply:
> Does [cmd/common/registry.go#L288-L296](https://github.com/inspektor-gadget/inspektor-gadget/blob/main/cmd/common/registry.go#L288-L296) need similar documentation too?

That is related to builtin gagets but good idea to include the comment. 

> And docs/reference/run.mdx too?

It seems it is already documented here: https://github.com/inspektor-gadget/inspektor-gadget/blob/main/docs/reference/run.mdx#examples